src/hooks/useENSRegistrationCosts.ts

```diff
@@ -309,9 +313,9 @@ export default function useENSRegistrationCosts({
       estimatedGasLimit,
       estimatedNetworkFee: formattedEstimatedNetworkFee,
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
     commitGasLimit,
-    gasFeeParams,
```

Comment:
this one updates often, doesn't make sense to update everything when this changes